galley/pkg/crd/validation/webhook.go

```diff
@@ -53,6 +53,10 @@ const (
 	watchDebounceDelay = 100 * time.Millisecond
 )
 
+func init() {
+	v1beta1.AddToScheme(runtimeScheme)
+}
```

Comment:
Can you move this init() function immediately below the related `var` declaration for runtimeScheme, etc.?

Reply:
ok

Reply:
done